### DISCUSSION TITLE
R68S随机自动重启

### DISCUSSION THREAD

**Discussion by Constantinefun** [coolsnowwolf/lede#9607] [R68S / 电犀牛专区]
R68S 4G版
docker版6.15固件
在未知情况下自动重启，不定时，不规律，重启后系统会报错：
Sat Jun 18 16:27:59 2022 daemon.err ttyd[5228]: [2022/06/18 16:27:59:6377] NOTICE: time discontiguity: at old time 1655540667s, new time 1655540879s: +212s

**Comment by daghy** [comment]
启动项里把mentohust禁用试试看吧。。Q群有人说过。这是锐捷上网。用不上的。

**Reply by dnslin** [reply]
那个人就是我 事实上删了也没用

**Comment by caoya000** [comment]
删掉mentohust后仍然会随机重启，并且重启前无日志记录

**Reply by dnslin** [reply]
确实 我也有这种情况 是serverchan 通知我才知道是重启 我一直以为是重新拨号

**Comment by kdatamr** [comment]
日志差不多内容，确实删掉锐捷依然会重启。等大雕修复吧。

**Comment by hww-png** [comment]
随机重启+1，无相关日志

**Comment by dnslin** [comment]
希望大雕能关注一下

**Reply by diy97** [reply]
随机重启+2

**Comment by martin0607** [comment]
随机重启+1
不知道是不是温度过高，最高60℃

**Reply by dnslin** [reply]
我50°

**Comment by puteulanus** [comment]
随机重启+1
一觉醒来看日志早上连续重启了两次，相隔半小时

**Reply by puteulanus** [reply]
7.1 docker 版运行两天多刚刚又自己重启了，先把 docker 停了观察一下

**Reply by puteulanus** [reply]
把 docker 停了到现在 uptime 10d 17h 了

**Comment by daghy** [comment]
我后面换成非docker的6.15版。就没有自动重启过了。。

**Reply by kdatamr** [reply]
兄弟怎么样了，非docker用了几天还算稳定吗？

**Comment by LidoCKT** [comment]
6.15docker版的问题 换别的没问题

**Comment by qianyimochen** [comment]
+1 基本上一天多就会重启一次

**Comment by shaufo** [comment]
随机重启+1，无相关日志

**Comment by daghy** [comment]
非docker的很稳定。6.15的运行连续3天没重启过。后面刷6.26了。看看这版情况。




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年7月3日(星期天) 上午10:49
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [coolsnowwolf/lede] R68S随机自动重启 (Discussion #9607)





 
兄弟怎么样了，非docker用的还算稳定吗？

**Reply by kdatamr** [reply]
我目前626docker 感觉重启更频繁了，24小时至少2-3次。温度53上下也不高，回头换非docker看下

**Comment by p-w-p** [comment]
7.1开机27s左右重启，人麻了

**Reply by p-w-p** [reply]
润回626非docker版了，挺稳的
可能是我不太会用docker吧

**Comment by AshakurasA** [comment]
最新版重启问题依旧存在，运行了7天，今天下午突然就重启了。

**Comment by ssmmtt** [comment]
8.22docker版自动重启，没有重启前错误日志，难受
